packages/map/src/types/MapConfig.ts

```diff
@@ -164,4 +164,6 @@ export type MapConfig = Visualization & {
   hexMap: HexMapSettings
   filterBehavior: string
   visual: MapVisualSettings
+  // version of the map
+  version: `${number}.${number}.${number}`
```

Comment:
Can we move this out to the core library and import it? everything will be using this

`type VersionNumber = \`${number}.${number}.${number}\``